images/tier1/bullet.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'd run this through https://jakearchibald.github.io/svgomg/ to clean it up and make sure you follow all the instructions in the README about sizes and such.